test/users.js

```diff
@@ -0,0 +1,167 @@
+'use strict';
+
+const Path = require('path');
+const { expect } = require('code');
+const Hapi = require('hapi');
+const Lab = require('lab');
+const StandIn = require('stand-in');
+const CloudApiGql = require('../lib/');
+const { CloudApi } = require('../lib/cloudapi');
+
+
+const lab = exports.lab = Lab.script();
+const it = lab.it;
+
+
+it('can get your account', async () => {
+  const user = {
+    id: '4fc13ac6-1e7d-cd79-f3d2-96276af0d638',
+    login: 'barbar',
+    email: 'barbar@example.com',
+    companyName: 'Example',
+    firstName: 'BarBar',
+    lastName: 'Jinks',
+    phone: '(123)457-6890',
+    updated: '2015-12-23T06:41:11.032Z',
+    created: '2015-12-23T06:41:11.032Z'
+  };
+
+  const keys = [{
+    name: 'test',
+    fingerprint: 'fingerprint',
+    value: 'foo'
+  }];
+
+  const server = new Hapi.Server();
+  StandIn.replace(CloudApi.prototype, 'fetch', (stand) => {
```

Comment:
Instead of nesting the `replace()` calls, can you do something alone these lines:

```js
StandIn.replace(CloudApi.prototype, 'fetch', (stand) => {
  if (stand. invocations === 1) {
    return user;
  }

  return keys;
}, { stopAfter: 2 });
```